unattended.sh

```diff
@@ -123,8 +123,11 @@ function log() {
 	# get the message log level
 	log_message_level="${1}"
 
-	# get the log message
-	log_message="${2}"
+        # shift once to get the log message (string or array)
+        shift
```

Comment:
use the same tabs as in the rest of the script please :)
```suggestion
	# shift once to get the log message (string or array)
	shift
```